docs/configuration.md

```diff
@@ -8,6 +8,7 @@ and variables needed to properly configure SC4S for your environment.
 | Variable | Values        | Description |
 |----------|---------------|-------------|
 | SC4S_USE_REVERSE_DNS | yes or no(default) | use reverse DNS to identify hosts when HOST is not valid in the syslog header |
+| SC4S_REVERSE_DNS_KEEP_FQDN | yes or no(default) | don't extract hostname from FQDN, pass the full domain name to HOST instead |
 | SC4S_CONTAINER_HOST | string | variable passed to the container to identify the actual log host for container implementations |
```

Comment:
| SC4S_CONTAINER_HOST | string | Variable that is passed to the container to identify the actual log host for container implementations. |